tests/ObjectEncryptorTest.php

```diff
@@ -945,7 +950,7 @@ public function testGetRegisteredWrapperEncryptors(
                 $encryptedBranchType,
                 'my-component',
                 'my-project',
-                ObjectEncryptor::BRANCH_TYPE_DEV,
+                ObjectEncryptor::BRANCH_TYPE_DEFAULT,
```

Comment:
Tady jsem to sjednotil, aby se dalo ověřit, že to jde všechno dekryptovat jedním encryptorem